src/tekton/triggertype.ts

```diff
@@ -68,20 +68,47 @@ export type PipelineModalFormResource = {
   resourceRef: string;
 };
 
+export type PipelineModalFormWorkspace = {
+  name: string;
+  type: string;
+  data:
+    | {
+        emptyDir: {};
+      }
+    | {
+        secret: VolumeTypeSecret;
+      }
+    | {
+        configMap: VolumeTypeConfigMaps;
+      }
+    | {
+        persistentVolumeClaim: VolumeTypePVC;
+      };
```

Comment:
This type definition a bit odd, can you rewrite this? Maybe with some intermediate types.

Reply:
I remove this type definition I was not using it Thanks